examples/k8s/hz-go-service/service.go

```diff
@@ -0,0 +1,347 @@
+package main
```

Comment:
Overall structure looks good, however it is kind of hard to navigate between config, cluster creation, route registration and handlers. Would it be possible to split this into two files?